plugins/feedback-backend/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## @janus-idp/backstage-plugin-feedback-backend [1.3.8](https://github.com/janus-idp/backstage-plugins/compare/@janus-idp/backstage-plugin-feedback-backend@1.3.7...@janus-idp/backstage-plugin-feedback-backend@1.3.8) (2024-06-05)
```

Comment:
this plugin is moved to the community repository. is it necessary to keep track of this packages

Reply:
@riginoommen I'm rebasing the 1.2.x branch with the latest changes in order to backport a number of CVE fixes (it's out of sync).  My understanding is the maintenance releases will continue to be done in the janus-idp repo.

Reply:
yeah looks like the earliest version in community-plugins is 1.7, so we need to maintain old stuff here until it's EOL in Dec with the release of RHDH 1.4.